Source/Scene/Cesium3DTileset.js

```diff
@@ -0,0 +1,1654 @@
+         */
+        maximumScreenSpaceError : {
+            get : function() {
+                return this._maximumScreenSpaceError;
+            },
+            set : function(value) {
+                //>>includeStart('debug', pragmas.debug);
+                Check.typeOf.number.greaterThanOrEquals('maximumScreenSpaceError', value, 0);
+                //>>includeEnd('debug');
+
+                this._maximumScreenSpaceError = value;
+            }
+        },
+
+        /**
+         * The maximum amount of memory in MB that can be used by the tileset.
+         * Tiles not in view are unloaded to enforce this.
+         * <p>
+         * If decreasing this value results in unloading tiles, the tiles are unloaded the next frame.
+         * </p>
+         * <p>
+         * If tiles sized more than <code>maximumMemoryUsage</code> are needed
+         * to meet the desired screen-space error, determined by {@link Cesium3DTileset#maximumScreenSpaceError},
+         * for the current view, then the memory usage of the tiles loaded will exceed
+         * <code>maximumMemoryUsage</code>.  For example, if the maximum is 256 MB, but
+         * 300 MB of tiles are needed to meet the screen-space error, then 300 MB of tiles may be loaded.  When
+         * these tiles go out of view, they will be unloaded.
+         * </p>
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {Number}
+         * @default 512
+         *
+         * @exception {DeveloperError} <code>maximumMemoryUsage</code> must be greater than or equal to zero.
+         * @see Cesium3DTileset#totalMemoryUsageInBytes
+         */
+        maximumMemoryUsage : {
+            get : function() {
+                return this._maximumMemoryUsage;
+            },
+            set : function(value) {
+                //>>includeStart('debug', pragmas.debug);
+                Check.typeOf.number.greaterThanOrEquals('value', value, 0);
+                //>>includeEnd('debug');
+
+                this._maximumMemoryUsage = value;
+            }
+        },
+
+        /**
+         * The tileset's bounding volume.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {TileBoundingVolume}
+         * @readonly
+         *
+         * @exception {DeveloperError} The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.
+         */
+        boundingVolume : {
+            get : function() {
+                //>>includeStart('debug', pragmas.debug);
+                if (!this.ready) {
+                    throw new DeveloperError('The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.');
+                }
+                //>>includeEnd('debug');
+
+                return this._root._boundingVolume;
+            }
+        },
+
+        /**
+         * The tileset's bounding sphere.
```

Comment:
Could be left over from a refactor.  Remove it if it is not currently used.  I don't think there is a good public API use case other than "zoom to tile", which is only useful for debugging at best.